src/bin/order_and_cancel.rs

```diff
@@ -15,24 +15,24 @@ async fn main() {
         .parse()
         .unwrap();
```

Comment:
Don't change this file. If you feel strongly that an example of placing a spot order should exist, then better to create a new binary (spot_order.rs maybe?)

Reply:
Got it, I've added a new example